tests/Dockerfile

```diff
@@ -74,7 +74,7 @@ RUN mkdir -p /usr/local/bin /home/user /build /secrets /cache/images /cache/gith
     ln -snf /secrets/zanata.ini /home/user/.config/zanata.ini && \
     git clone https://github.com/cockpit-project/cockpit /build/cockpit && \
     cd /build/cockpit && npm install && \
-    chown -R user /build /secrets /cache /home/user
+    chmod -R ugo+w /build /secrets /cache /home/user
```

Comment:
This should not have any effect for /cache and /secrets as get bind-mounted from the host/from a volume (even in your new cockpit-tasks-restricted.json). These directories should be completely empty during container creation.

/home/user is not a volume, but why does the  home dir need to be world-writable? Shouldn't the original `user` ownership of it be enough?

Reply:
When run in Openshift, the pod is run as a random uid. This means we don't have permissions to write to those directories. In order to cover *both* the bind-mounted use case and the random-uid use case this is a way to solve this.

The ```user``` account does not play in Openshift at all.

Reply:
I understand the random uid and the permission problem. I don't understand how this chown command has any visible effect. This happens on the *unmounted* mount point directory during container build, doesn't  it? Once a bind mount is done onto that dir, the original permissions don't matter at all - what's relevant is the permissions of the root directory of the *mounted* device. I. e. what matters is the permissions of the files in the mounted volume, not of this mount point.

Reply:
Bwah, of course this is because it is now *chmod*, not *chown*. "Wer lesen kann, ist klar im Vorteil!" Sorry! :-)